frontend/vue/components/UserAccount/UserAccountLayout.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div class="user-account">
+    <nav class="user-account__section-nav">
+      <div class="user-account__section-nav__user-data">
+        <span class="user-account__section-nav__user-data__name">{{ userName }}</span>
+        <span class="user-account__section-nav__user-data__role">{{ userRole }}</span>
+      </div>
+      <div class="user-account__section-nav__link-list">
+        <AppLink
+          v-for="{displayName, hash} in sectionList"
+          :key="hash"
+          class="user-account__section-nav__link-list__link"
+          :class="{'user-account__section-nav__link-list__link_active': activeSection === hash}"
+          :url="hash"
+          target="_self"
+        >
+          {{ displayName }}
+        </AppLink>
+        <AppLink
+          class="user-account__section-nav__link-list__link"
+          url="/LogOut"
```

Comment:
As I understand, this is a placeholder. Nevertheless, it renders `<a href="false">`.

To avoid confusion, I'd suggest:

```suggestion
          url="#"
```